### PR TITLE
Concat files utility file

### PR DESCRIPTION
**Pull request by alexbarrera**

### CODE REVIEW

**Comment by alexbarrera** [comment]
Added samtools support (version 1.3)

File: workflow-utils/bin/concat-files.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cat $1 $2
```
Comment (dleehr) [inline workflow-utils/bin/concat-files.sh]: Assuming this is for a cwltool, it seems unnecessary. Can't the tool just specify `cat` as its baseCommand and take two inputs?

Comment (alexbarrera) [inline workflow-utils/bin/concat-files.sh]: Very true. I just removed the file.

**Comment by dleehr** [comment]
I'll need to set up automated builds on docker hub to make these into images, but I'm assuming you can build them yourself now

**Comment by alexbarrera** [comment]
Right, I'm building them in local for now.

**Pull request merged**